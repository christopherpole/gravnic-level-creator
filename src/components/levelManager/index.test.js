import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import LevelManager from './index';

configure({ adapter: new Adapter() });

it('renders without exploding', () => {
  const levelManager = shallow(<LevelManager />);

  expect(levelManager).toHaveLength(1);
});

it('matches the current snapshot', () => {
  const levelManager = shallow(<LevelManager />);

  expect(toJson(levelManager)).toMatchSnapshot();
});
