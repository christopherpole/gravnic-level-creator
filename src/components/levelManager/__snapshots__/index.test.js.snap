// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the current snapshot 1`] = `
<styled.section>
  <p>
    Level Manager goes here
  </p>
</styled.section>
`;
